kafka-samples/kstreams-log-user-clicks-per-region/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.1.3.RELEASE</version>
+  </parent>
+
+  <groupId>org.springframework.cloud.stream.app</groupId>
+  <artifactId>kstreams-log-user-clicks-per-region</artifactId>
+  <version>1.0.0.BUILD-SNAPSHOT</version>
+  <name>kstreams-log-user-clicks-per-region</name>
+
+
+  <dependencies>
+    <!-- This dependency adds Spring Cloud Stream Binder for Apache Kafka -->
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-starter-stream-kafka</artifactId>
+      <version>2.1.1.RELEASE</version>
```

Comment:
hardcoded dependency. Might want to consider using a BOM to manage the versions. A bit easier that way to maintain each time we update the boot parent.